regression-tests/README.md

```diff
@@ -29,7 +29,13 @@ hurl --test --jobs 1 --repeat 1 --variable BASE_URL="https://api.quality.ohsome.
 
 ### Release 1.10
 
-| indicator                    | `--jobs=1 --repeat 1` | `--jobs 15 --repeat 10` | `--jobs 15 --repeat 100` |
-| ---                          | ---                   | ---                     | ---                      |
-| land-cover-thematic-accuracy | 15547 ms              | 18217 ms                | 134782 ms                |
-| land-cover-completeness      | 29146 ms              | 45915 ms                | 386117 ms                |
+| indicator                      | `--jobs=1 --repeat 1` | `--jobs 15 --repeat 10` | `--jobs 15 --repeat 100`          |
+|--------------------------------|-----------------------|-------------------------|-----------------------------------|
+| land-cover-thematic-accuracy   | 15547 ms              | 18217 ms                | 134782 ms                         |
```

Comment:
another small comment: it might be good to report in seconds here and not using ms. This would make these numbers easier to read. (and at the moment seconds will still be a good enough unit to distinguish the performance for the various indicators)